server/mix.exs

```diff
@@ -68,6 +68,7 @@ defmodule BuffServer.MixProject do
        [env: :prod, hex: "cowboy", repo: "hexpm", optional: false, override: true]},
       {:cowlib, "~> 2.7.3",
        [env: :prod, hex: "cowlib", repo: "hexpm", optional: false, override: true]},
+      {:gun, "~> 1.3", override: true},
```

Comment:
hack to make integration tests work.. 
Opened an issue to fix:
https://github.com/elixir-grpc/grpc/issues/102